showpass/Showpass-E2E-tests
# TA-102 first commit

**Description:**
Any event can be configured as a free admission event. There is no need to purchase tickets. As such, this user story will be verifying that such an event can be created and verified on the front-end.

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-102

**How to test:**
Run the following spec and ensure all tests pass cypress/e2e/checkout/checkout.cy.js

**Test execution results:**
[Cypress Cloud results](https://cloud.cypress.io/projects/e9axi7/runs/186/test-results?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%5D&statuses=%5B%5D&testingTypesEnum=%5B%5D)
